MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1083,27 +1083,50 @@ public bool CanStreamCopyAudio(EncodingJobInfo state, MediaStream audioStream, s
         {
             var bitrate = request.VideoBitRate;
 
-            // If specific values were requested, then force the caller to supply a bitrate as well
-            if (request.Height.HasValue && request.Width.HasValue)
-            {
-                return bitrate;
-            }
-
             if (videoStream != null)
             {
-                if (bitrate.HasValue)
-                {
-                    var inputVideoCodec = videoStream.Codec;
-                    bitrate = ScaleBitrate(bitrate.Value, inputVideoCodec, outputVideoCodec);
+                var isUpscaling = request.Height.HasValue && videoStream.Height.HasValue &&
+                                   request.Height.Value > videoStream.Height.Value && request.Width.HasValue && videoStream.Width.HasValue &&
+                    request.Width.Value > videoStream.Width.Value;
 
-                    // If a max bitrate was requested, don't let the scaled bitrate exceed it
-                    if (request.VideoBitRate.HasValue)
+                // Don't allow bitrate increases unless upscaling
```

Comment:
What does this mean exactly?

Reply:
Well, that's something that Emby 3.5.3 also does. If it is working correctly, it should prevent the bitrate from getting 2 or 2.5 times bigger than the original (those are the possible factors that the function GetMinBitrate will apply), in case it is not upscaling, because that would be a waste. On the contrary, it may be necessary to increase the bitrate beyond those values if the video is upscaled to a much bigger size, so in that case the restriction is not applied.